java/src/jmri/jmrit/logixng/SymbolTable.java

```diff
@@ -176,7 +176,9 @@ public VariableData(
                 String initialValueData) {
             
             _name = name;
-            _initalValueType = initalValueType;
+            if (initalValueType != null) {
+                _initalValueType = initalValueType;
```

Comment:
Would it be hard to fix the spelling of these two variables?  (initialValueData has the right one)

Reply:
Thanks. Fixed.

The part of the panel xml file that the class DigitalCallModuleXml reads and writes is not updated. It has the element `initalValueType`. Fixing that is easy but requires that the old name is still valid for a while. I don't want to do that change so close to the test release.